### PR TITLE
Fix Flex class docs by replacing 'vertical space' with 'space on its main axis' so that the language fits both column and row.

### PR DESCRIPTION
**Pull request by clocksmith**
## Description

Fix Flex class docs by replacing 'vertical space' with 'space on its main axis' so that the language fits both column and row.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes tests for *all* changed/updated/fixed behaviors (See [Test Coverage]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -3548,8 +3548,8 @@ class PositionedDirectional extends StatelessWidget {
 /// horizontal) or [Column] (if it's vertical) instead, because that will be less
 /// verbose.
 ///
-/// To cause a child to expand to fill the available vertical space, wrap the
-/// child in an [Expanded] widget.
+/// To cause a child to expand to fill the available space on its main axis,
```
Comment (HansMuller) [inline packages/flutter/lib/src/widgets/basic.dart]: on its main axis sounds a bit like the child's axis

Maybe:
to fill the available space in the [direction] of this widget's main axis,

**Review by HansMuller — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/basic.dart (hunk 1 above)
Comment (clocksmith) [reply packages/flutter/lib/src/widgets/basic.dart]: You're right, "its" was ambiguous. Updated.

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -3548,8 +3548,8 @@ class PositionedDirectional extends StatelessWidget {
 /// horizontal) or [Column] (if it's vertical) instead, because that will be less
 /// verbose.
 ///
-/// To cause a child to expand to fill the available vertical space, wrap the
-/// child in an [Expanded] widget.
+/// To cause a child to expand to fill the available space in the [direction]
```
Comment (johnsonmh) [inline packages/flutter/lib/src/widgets/basic.dart]: I don't think `direction` is a variable/in-scope identifier. Should probably just be direction or \`direction`

**Review by johnsonmh — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/basic.dart (hunk 2 above)
Comment (HansMuller) [reply packages/flutter/lib/src/widgets/basic.dart]: https://docs.flutter.io/flutter/widgets/Flex/direction.html

**Review by willlarche — APPROVED** [review]

**Pull request merged**